ca/certificate-authority_test.go

```diff
@@ -96,6 +96,24 @@ var (
 	// Edited signature to become invalid.
 	WrongSignatureCSR = mustRead("./testdata/invalid_signature.der.csr")
 
+	// CSR generated by Go:
+	// * Random public key
+	// * CN = not-example.com
+	// * Includes an extensionRequest attribute for a well-formed TLS Feature extension
+	MustStapleCSR = mustRead("./testdata/must_staple.der.csr")
+
+	// CSR generated by Go:
+	// * Random public key
+	// * CN = not-example.com
+	// * Includes an extensionRequest attribute for an empty TLS Feature extension
+	TLSFeatureUnknownCSR = mustRead("./testdata/tls_feature_unknown.der.csr")
```

Comment:
There should be one more test case: TLS feature extension OID, but the value didn't match the must staple value.

It may be worthwhile to factor the test function out into a set of testCase structs, as we do elsewhere in the codebase. Up to you, though.